### ISSUE TITLE
Directory not Found

### ISSUE THREAD

**Issue by l0s3rr** [Chocobozzz/PeerTube#1375]
Good morning, are you okay?

Congrats for this job, i have a question, i have tried to install PeerTube on CentOS7, but this directory ""https://github.com/Chocobozzz/PeerTube/releases/download/${VERSION}/peertube-${VERSION}.zip " dosn't exist anymore?

Your application is omologated on CentOS7?

Anyway, thanks for attention. ;)

**Comment by Nutomic** [comment]
Did you set the environment variable for $VERSION?

**Comment by l0s3rr** [comment]
Yes, just in one Session, but in the Github dosn't exist anymore...

**Comment by l0s3rr** [comment]
I already exported now this variable and i got this download, thanks anyway.
